### PR TITLE
Blockies added

### LINKED ISSUE #4: Create Ethereum blockies for every address
Using [this](https://github.com/ethereum/blockies#readme) we should create blockies for users to identify etheruem addresses pictorially for enhancing security. This blockie should be visible with the address in the top nav bar or sidebar.

### PR DESCRIPTION
**Pull request by iamjaspreetsingh**
Fixes #4 added blockies at bottom left of home-page

### CODE REVIEW

**Comment by vaibhavchellani** [comment]
create PR to `contract-flow` as you forked from it

**Comment by iamjaspreetsingh** [comment]
So, i need to close this pr and create new one? Right?

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "Jiffy Web",
   "author": "Shubham Singh <imshubhamsingh@gmail.com>",
-  "license": "GNU",
```
Comment (vaibhavchellani) [inline package.json:6]: why ?

Comment (iamjaspreetsingh) [reply package.json:6]: i have not changed these files, may be auto generated

**Comment by iamjaspreetsingh** [comment]
i am closing this pr, will make new one..

**Pull request closed without merging**